sale_order_partner_invoice_period/views/res_partner_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_partner_form_inherit" model="ir.ui.view">
+        <field name="name">res.partner.view.form.inherit</field>
```

Comment:
Preserve the same name as original view and add ` (in sale_order_partner_invoice_period)` , so inspecting name we can figure out its origin.